src/app/(test)/yj/_components/OptionWithButtons.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+
+import { Button, Dropdown } from '@/components';
+import { DeleteIcon, MinusIcon, PlusIcon } from '@/public/index';
+import type { OptionTypes } from '@/types/ProductTypes';
+import classNames from 'classnames/bind';
+import { useState } from 'react';
+import styles from './ProductDetail.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface OptionCountProps {
+  optionText?: string;
+  price?: number;
+}
+
+interface OptionWithButtonProps {
+  optionList: OptionTypes[];
+  price: number;
+}
+
+type OperationType = 'plus' | 'minus';
+
+function OptionContainer({ optionText, price }: OptionCountProps) {
+  const [count, setCount] = useState<number>(1);
+  const [isOptionSelected, SetIsOptionSelected] = useState<boolean>(true);
```

Comment:
S 소문자로 변경하면 좋을 것 같습니다!
```suggestion
  const [isOptionSelected, setIsOptionSelected] = useState<boolean>(true);
```